mikeio1d/result_reader_writer/filter/filter.py

```diff
@@ -27,6 +27,8 @@ def use_filter(self) -> bool:
 
     def apply(self, result_data: ResultData):
         """Apply filter."""
+        if not self.use_filter():
```

Comment:
In terms of naming we added a lot of places Result, like ResultNetwork, etc. I think it makes sense to do the same for filter and in such a way we do not get a conflict with MIKE 1D.